### ISSUE TITLE
Add species diversity map views

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3041]
**Add species diversity map views**

It may be a good idea to replace or complement default observation density map views for taxa with species diversity maps, see Fig. 2 https://www.researchgate.net/publication/319273816_Identifying_global_hotspots_of_agricultural_and_tree_biodiversity_and_poverty_for_the_strategic_sharpening_of_Bioversity%27s_priority_research_areas/figures?lo=1.

This would highlight the biodiversity hotspots of the planet (instead or together with peak regions of observational activity). The content for species diversity maps would come from N unique species with at least 1 observation per grid cell within the selected higher taxon (skipping all taxa in between). Perhaps the hexagon tool could be reworked into that.

-----

Github user: @dschigel
User: [See in registry](https://www.gbif.org/api/feedback/user/34f1deaee068f9b969bf8d2a749f7774:c71a3dd51f80eec4cef4c91f1c76766c4721340fa643e982f45036129cb7a97cb862d72d9eeef7883a6577c1f94060525eaecea3e1a1d107f968c477b931aa36)
System: Chrome 85.0.4183 / Windows 10.0.0
Referer: https://www.gbif.org/species/search?q=
Window size: width 1280 - height 711
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-10-08T04:56:55.525Z',mode:absolute,to:'2020-10-08T05:02:55.525Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-10-08T04:56:55.525Z',mode:absolute,to:'2020-10-08T05:02:55.525Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: CRITICAL

**Comment by jhnwllr** [comment]
I totally agree that species count maps would be a very useful feature. I am not sure what would be the backend burden of adding them though.

**Comment by dschigel** [comment]
@jhnwllr if you are genuinely interested, shall we try to dream up what would such map show and why we would needed, and then offer the idea to budget resources and tech needed to implement? I like dreaming unrestricted, but I am also ok to check feasibility first, too.

**Comment by timhirsch** [comment]
I agree in principle but we would need to be very careful about what we would be showing with such maps: as @jhnwllr demonstrated in one of his blog posts, unprocessed 'species diversity' maps would still correlate with sampling effort and result in perverse implications such as Belgium being a biodiversity hotspot. Given the variety of efforts currently under way to create species richness maps and methods of determining areas of high biodiversity value, we'd need to be completely clear what such a view would add - and certainly not expect it to identify 'biodiversity hotspots' which are defined by a combination of species richness and the level of habitat loss.

**Comment by jhnwllr** [comment]
maps dream wish list might be: 
1. Range Polygon Overlays
2. Species Counts
3. Data type color highlighting (basis of record ect)

**Comment by dschigel** [comment]
A good way to show the gaps / data potential is to move from what is there (N data points, N species), to what could be / expected to be there. With digitization potential maps one needs to have enough numeric content in the catalogue of collections and other holders, but for estimated species numbers for countries we need an external source, and at the latest check and discussion with @ahahn-gbif there is none. Maps showing approach to the targets (of better representation) is a slightly more positive way to show data gaps / data hungers, and heat mapping could be used for prioritizing data mobilization efforts and resources spatially. NN digitally known out of MM potential. Estimates of MM should be enough. I would say breaking down by BoR is nice, but can be done later, and also BoR vocab needs to be tidied up before BoR are resurfaced at maps.
